### ISSUE TITLE
Wrong background image is used for a subtree with split

### ISSUE THREAD

**Issue by joonro** [yjwen/org-reveal#263]
Hi,

It seems when I have two subtrees with `:REVEAL_BACKGROUND:`, and if the second subtree has `#+REVEAL: split` in it, then the third slide uses the background image from the first subtree, not its own.

Simple example: 

```org
* First Slide
:PROPERTIES:
:REVEAL_BACKGROUND: /Path/To/Image1.jpg
:END:

First slide
* Second Slide
:PROPERTIES:
:REVEAL_BACKGROUND: /Path/To/Image2.jpg
:END:

Second slide

#+REVEAL: split

Third slide
```

In my machine (`GNU Emacs 25.1.1 (x86_64-w64-mingw32)`, org 9.0.3), the third slide has `Image1.jpg` instead of `Image2.jpg` as its background.

**Comment by lechten** [comment]
The root cause is that `org-reveal--last-slide-section-tag` contains information for the _previous_ slide, not the one to be split.

**Comment by yjwen** [comment]
Issue has been fixed by 83b8d9f .
